payment/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
 
-# Create your views here.
+from payment.models import Payment
+from payment.serializers import PaymentSerializer, PaymentListSerializer
+
+
+class PaymentViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Payment.objects.select_related("borrowing__book", "borrowing__user")
+
+    def get_serializer_class(self):
```

Comment:
Please, use same code style in project, for example:
```
    queryset = Borrowing.objects.all().select_related("book", "user")
    serializer_class = BorrowingSerializer

    def get_serializer_class(self):
        if self.action == "list":
            return BorrowingListSerializer
        elif self.action == "retrieve":
            return BorrowingDetailSerializer
        return self.serializer_class
```

Reply:
Resolved